c2cwsgiutils/acceptance/composition.py

```
@@ -86,9 +99,7 @@ def stop_all(self) -> None:
             os.makedirs(target_dir, exist_ok=True)
             for path in self.coverage_paths:
                 try:
-                    subprocess.check_call(  # nosec
-                        ["docker", "cp", path, target_dir], stderr=subprocess.STDOUT
-                    )
+                    subprocess.run(["docker", "cp", path, target_dir], check=True, timeout=60)  # nosec
```

Why removing stderr redirection?
Why not simply adding timeout=60 to `check_call`?

---

Because of the first sentence in the doc: https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module
Because we get the unwanted stderr...
My dream is to completely remove this file, I just get issue and hidden things with that way to do...

---

But then, if something goes wrong the output is silently drop, isn't it?

Do you have an example or more insights about the issues and hidden things you get?

> My dream is to completely remove this file, I just get issue and hidden things with that way to do...
Do you mean the composition mechamism?